examples/energy_mng/src/main.rs

```diff
@@ -32,11 +32,11 @@ pub fn main() {
 
     let window = MainWindow::new().unwrap();
 
-    let header_timer = header::setup(&window);
+    // let _ to keep the timer alive.
+    let _ = header::setup(&window);
```

Comment:
_ don't keep it alive

Reply:
Does the timer works also if the timer object is dropped?

Reply:
I think not

Reply:
@ogoffart this looks like the timer will be dropped:

```
impl Drop for Timer {
    fn drop(&mut self) {
        if let Some(id) = self.id.get() {
            let _ = CURRENT_TIMERS.try_with(|timers| {
                timers.borrow_mut().remove_timer(id);
            });
        }
    }
}
```

Reply:
I checked it. Without the timer will be dropped, even `let _ = ` does not keep the timer alive it needs to be `let _timer =`